sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/ResponseTypesTests.cs

```diff
@@ -31,14 +32,14 @@ public enum ResponseTypes
 
         /// <summary>Test the response value setting based on the response type expected.</summary>
         /// <param name="responseType">The response type to test as a function return value.</param>
-        [Theory]
-        [InlineData(ResponseTypes.String)]
-        [InlineData(ResponseTypes.HttpResponse)]
-        [InlineData(ResponseTypes.HttpResponseMessage)]
-        [InlineData(ResponseTypes.Unknown)]
-        [InlineData(ResponseTypes.AuthEventResponse)]
+        [Test]
+        [TestCase(ResponseTypes.String)]
+        [TestCase(ResponseTypes.HttpResponse)]
+        [TestCase(ResponseTypes.HttpResponseMessage)]
+        [TestCase(ResponseTypes.Unknown)]
+        [TestCase(ResponseTypes.AuthEventResponse)]
         [Obsolete]
-        public async void Tests(ResponseTypes responseType)
+        public async Task Tests(ResponseTypes responseType)
```

Comment:
We are we marking a test as obsolete?   Why not just remove the test?

Reply:
The obsolete tags here allows us to use obsolete methods and functions for tests, not obsolete the tests for further use.